apps/barista/src/pages/search-page/search-page.html

```diff
@@ -0,0 +1,32 @@
+<div class="ba-page-grid">
+  <main class="ba-page-main" id="main" role="main">
+    <article class="ba-search-page-content">
+      <h1>Search Barista</h1>
+      <dt-form-field class="ba-search-input">
+        <dt-label i18n>Search posts</dt-label>
+        <input type="text" dtInput #queryInput [(ngModel)]="queryValue" />
+      </dt-form-field>
+
+      <div
+        *ngFor="let result of (_searchResults$ | async)"
+        class="ba-search-result-item"
+      >
+        <div>
+          <a class="dt-link dt-external" [href]="result.url"
+            >{{ result.title }}</a
+          >
+          <p class="ba-search-contentextractions" *ngIf="result.contentMatches">
+            <span
+              dt-highlight
+              [term]="queryValue"
+              *ngFor="let contentMatch of result.contentMatches"
```

Comment:
NIT: you can have here a lot of occurrences so I would recommend using the trackedBy from ng-for

https://netbasal.com/angular-2-improve-performance-with-trackby-cc147b5104e5

Reply:
It's limited on the server side to 5 occurrences, I don't think we need a trackBy here.